globals.js

```diff
@@ -0,0 +1,22 @@
+/* eslint import/no-commonjs: off, flowtype/require-valid-file-annotation: off, flowtype/require-return-type: off */
+
+const postRobotGlobals = require("@krakenjs/post-robot/globals");
```

Comment:
@faiz121 was this needed? Why was this added? Sorry for the naive question here. I believe we do need some of it but not all of it. I was hoping you can share some details around why you put in the values you did so I can better understand.